client/reader/following-edit/index.jsx

```diff
@@ -34,7 +34,7 @@ import FollowingImportButton from './import-button';
 import FeedDisplayHelper from 'reader/lib/feed-display-helper';
 import SectionHeader from 'components/section-header';
 import Button from 'components/button';
-const stats = require( 'reader/stats' );
+import stats from 'reader/stats';
```

Comment:
I've been staring at this for a while, and I think the only surefire way to catch all these kinds of errors in the codebase would be to enhance our code linter to resolve modules (ex. [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import)). 

I'll follow-up with what I find out!